cli/tests/unit_node/os_test.ts

```diff
@@ -252,28 +253,39 @@ Deno.test({
 });
 
 Deno.test({
-  name: "APIs not yet implemented",
+  name: "os.setPriority() & os.getPriority()",
+  // disabled because os.getPriority() doesn't work without sudo
+  ignore: true,
```

Comment:
**suggestion:** Just to be extra cool maybe only disable this if user ID is non-zero? ie. When not being run as sudo.